requests/hooks.py

```diff
@@ -11,12 +11,16 @@
 ``response``:
     The response generated from a Request.
 """
-HOOKS = ['response']
+
+RESPONSE_HOOK = 'response'
+
+HOOKS = [RESPONSE_HOOK, ]
```

Comment:
There's no need for `, ` here. It should just be `[RESPONSE_HOOK]` (if we even do this)